libs/tup-components/src/projects/ProjectsTable.tsx

```diff
@@ -4,10 +4,8 @@ import { LoadingSpinner, Message } from '@tacc/core-components';
 import { ProjectTitle, PrinInv, Allocations } from './ProjectsCells';
 import { ProjectsRawSystem, useProjects } from '@tacc/tup-hooks';
 import styles from './Projects.module.css';
```

Comment:
In either this PR or an independent one, please remove this import **and** that stylesheet, because:

- Its styles will out of date (after #70).
- The stylesheet name is inconsistent.[^1]

[^1]: If ProjectsTable needed styles, they should be in a `ProjectsTable.module.css`. Examples of styles such a stylesheet could use are in ["O Table Wrap: Flexible, Horz." example client CSS](https://dev.tup.tacc.utexas.edu/static/ui/components/raw/o-table-wrap/extra.flexible-horz.css) ([source](https://dev.tup.tacc.utexas.edu/static/ui/components/detail/o-table-wrap--flexible-horz.html)) and ["O Table Wrap: Overflow Scroll" example client CSS](https://dev.tup.tacc.utexas.edu/static/ui/components/raw/o-table-wrap/extra.overflow-scroll.css) ([source](https://dev.tup.tacc.utexas.edu/static/ui/components/detail/o-table-wrap--overflow-scroll.html)).

Reply:
Thanks for looking this over. Very helpful! If you think it would better down the line to change the divs in the Dashboard page to <body>,  just let me know and I can add that.